### PR TITLE
add firehose.hackclub.com

### PR DESCRIPTION
**Pull request by radioblahaj**
# [Adding/Deleting/Updating] `subdomain.[hackclub.com/dino.icu/etc]`

## Description

Provide a description here of the changes, and if adding a subdomain provide a rationale & what it will be used for.

### CODE REVIEW

**Review by saumilthecode — COMMENTED** [review]
is this intentional lol

**Comment by saumilthecode** [comment]
^ the edit to _github-pages-challenge-arsenic-club.arsenic:

**Comment by radioblahaj** [comment]
oops

**Comment by radioblahaj** [comment]
fixed

File: hackclub.com.yaml [hunk 1]
```diff
@@ -432,6 +432,10 @@ forge:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+firehose:
+  - ttl: 600
+    type: CNAME
+    value: cylindrical-rose-bklwrtred01ytycfprxq1lvm.herokudns.com
```
Comment (alx-alexpark) [inline hackclub.com.yaml:438]: Dont forget the trailing dot!
```suggestion
    value: cylindrical-rose-bklwrtred01ytycfprxq1lvm.herokudns.com.
```

**Review by alx-alexpark — CHANGES_REQUESTED** [review]

**Review by dispherical — APPROVED** [review]

**Pull request merged**